blkin-lib/zipkin_c.h

```diff
@@ -111,6 +111,26 @@ typedef enum {
 } blkin_annotation_type;
 
 /**
+ * @typedef blkin_core_annotation
+ * Core annotations used by Zipkin used to denote the beginning and end of 
+ * client and server spans.
+ * For more information refer to
+ * https://github.com/openzipkin/zipkin/blob/master/zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinCore.thrift
+ */
+typedef enum {
+	CLIENT_SEND=1,
+	CLIENT_RECV,
+	SERVER_SEND,
+	SERVER_RECV,
+	WIRE_SEND,
+	WIRE_RECV,
+	CLIENT_SEND_FRAGMENT,
+	CLIENT_RECV_FRAGMENT,
+	SERVER_SEND_FRAGMENT,
+	SERVER_RECV_FRAGMENT
+} blkin_core_annotation;
```

Comment:
i like the way that openzipkin does these annotations, i.e. `const string CLIENT_SEND = "cs"`. i'd prefer that we use a string representation too, so that we don't need the extra step to map enum values into strings with `get_core_annotation_value()`. could you change these to string constants?
```c
const char* const CLIENT_SEND = "cs";
const char* const CLIENT_RECV = "cr";
...
```